troubleshoot-install.md

```diff
@@ -50,7 +50,21 @@ If you don’t see a name, you need to add a storage class. For help doing this,
 * AWS: [https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
 * Azure: [https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk](https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-disk)
 
-Alternatively, you can [deploy Kubecost without persistent storage](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml#L148).
+Alternatively, you can deploy Kubecost without persistent storage to store by following these steps:
+
+> Note: This setup is only for experimental purpose. The metric data is reset when kubecost's pod is rescheduled.
```

Comment:
Potential Future TODO: we may want to reference our new backup & restore feature here... i.e. data is only reset when external storage isn't enabled.

Reply:
Noted. Thank you for your feedback Webb, I will try new feature and update the doc accordingly in new PR.